FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -130,9 +132,22 @@ public override void OnFixedUpdate()
                 if (!bIsEnabled)
                 {
                     strCollectingStatus = "Disabled";
-                    strStarDist = (CalculateDistanceToSun() / 1000).ToString("F2") + " km"; // trying to solve the bullshit
+                    strStarDist = UpdateDistanceInGUI(); // passes the distance to the GUI
+                    return;
+                }
+
```

Comment:
Well it doesn't have to be exact, but a magnetosphere only the size of the atmosphere seems a bit limited, but we could use it as the point at which the magneto strength is at it strongest then gradually make it weaker the further from the edge of the atmosphere. A function like MagnetoSphereStrenthRatio = vessel.altitude < atmosphere.height * 10 ?  vessel.altitude / atmosphere.height : 0